systemtests/src/test/java/io/enmasse/systemtest/bases/TestBase.java

```diff
@@ -81,15 +82,19 @@ protected static void deleteAddressSpace(AddressSpace addressSpace) throws Excep
         }
     }
 
+    protected static void deleteAllAddressSpaces() throws Exception {
+            TestUtils.deleteAllAddressSpaces(addressApiClient, logCollector);
+    }
```

Comment:
I think you need to use wait method for waiting for deletion  all addressspaces.